superset/reports/commands/execute.py

```diff
@@ -640,11 +633,13 @@ class AsyncExecuteReportScheduleCommand(BaseCommand):
     - On Alerts uses related Command AlertCommand and sends configured notifications
     """
 
-    def __init__(self, task_id: str, model_id: int, scheduled_dttm: datetime):
+    def __init__(
+        self, task_id: Union[UUID, str], model_id: int, scheduled_dttm: datetime
```

Comment:
Any reason we support both `str` and `UUID` and not just the later?

Reply:
Because the unittest generates a UUID directly and I didn't want to convert it to string then to uuid again... but you are right, I shouldn't make the app code more complex just to accommodate testing.